docs/vhlo.md

```diff
@@ -71,3 +71,76 @@ frameworks (producers) only need to target StableHLO ops, and compiler
 backends (consumers) only need to support the latest version, which is the
 StableHLO op set. Conversions to and from VHLO are taken care of with machinery
 maintainted in the StableHLO repo.
+
+## Contributing Incompatible Changes
+
+All changes with compatibility implications must go through the RFC process.
+This includes adding, deprecating, or renaming a feature. Once the RFC is
+approved, here are some contribution guidelines:
+
+### Bump the Version Number in Version.h
+
+Prior to updating VHLO ops, attribute, types, or conversions, increment the
+minor version number in [Version.h](https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/Version.h).
+Any new VHLO features added would use this bumped version, for example after
+bumping `0.10.0 --> 0.11.0`, a new op in [VhloOps.td](https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/VhloOps.td)
+would use:
+
+```tablegen
+VHLO_Op<"abs_v2", "0.11.0", "current">
+```
+
+### Add Required VHLO Implementation and Conversions
+
+The exact code needed to integrate a new feature will vary, but for the most
+part the following will need to change:
+
+* For all changes:
+  * Update the version log in [VhloDialect.td](https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/VhloDialect.td#L29)
+* For new ops:
+  * Add the op in [VhloOps.h](https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/VhloOps.h)
```

Comment:
VhloOps.td? (Same below).

Reply:
I don't think this has been addressed. Maybe forgot to commit and/or push?

Reply:
Ah. This exact issue occurred twice, I fixed the other one previously. Fixed both now, thanks